### ISSUE TITLE
Files synced with desktop client not readable in browser - encryption issue?

### ISSUE THREAD

**Issue by sndrsk** [owncloud/client#3554]
### Steps to reproduce
1. Sync file via ownCloud desktop client into an external storage (Google Drive)
2. Attempt to open the file in ownCloud in browser

### Expected behaviour
File should open/download

### Actual behaviour
File cannot be read via ownCloud in the browser. If viewed from ownCloud, it opens as if it's a text file and starts with:

%PDF-1.4
%Çì¢
5 0 obj
<</Length 6 0 R/Filter /FlateDecode>>
stream

and then a bunch of garbage.

If I try to view it from Google Drive, it starts with HBEGIN:oc_encryption_module:OC_DEFAULT_MODULE:cipher:AES-256-CFB:HEND and then a bunch of garbage.

### Server configuration
**Operating system**: Linux

**Web server:** Debian

**Database:** MySQL

**PHP version:** 5.4

**ownCloud version:** (see ownCloud admin page) 8.1

**Updated from an older ownCloud or fresh install:** Fresh

**List of activated apps:** Activity, Calendar, Default encryption module, Deleted files, External sites, File locking, First run wizard, Mail template editor, PDF viewer, Pictures, Provisioning API, Share files, Text editor, Updater, Versions, Video viewer, Bookmarks, Contacts, Mail, News, Tasks, OwnNote

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system --public
from within your ownCloud installation folder

or 

Insert your config.php content here
(Without the database password, passwordsalt and secret)
```

**Are you using external storage, if yes which one:** local/smb/sftp/... Yes, Google Drive and Dropbox

**Are you using encryption:** Yes

**Are you using an external user-backend, if yes which one:** No

### Client configuration
**Browser:** Google Chrome 44.0.2403.125 m

**Operating system:** Windows 7 64-bit

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log 
c) ...
```

**Comment by ogoffart** [comment]
So if in understand correctly, you cannot open the file from the owncloud web interface.
But can you download the file again with another desktop client?

**Comment by guruz** [comment]
Do you find a similar bug in https://github.com/owncloud/core ? This tracker is for the client.

**Comment by sndrsk** [comment]
@ogoffart Correct on both questions. I just verified that if I place a file in the folder to be synced running the client on Windows 7, it will sync and can be opened by the client on Windows 10. However I cannot open it from the owncloud web interface. I have tested PDF, jpg, and txt.

@guruz I have not found a similar bug in core. Since I can upload via the web interface just fine and read files via the web interface I thought the client tracker would have been a good start.
